README.md

```diff
@@ -3,3 +3,33 @@
 [![pre-commit hooks](https://github.com/AlexMomot-717/python-project-83/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/AlexMomot-717/python-project-83/actions)
 [![Maintainability](https://api.codeclimate.com/v1/badges/7ad5b8e6d1a22a4fdf34/maintainability)](https://codeclimate.com/github/AlexMomot-717/python-project-83/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/7ad5b8e6d1a22a4fdf34/test_coverage)](https://codeclimate.com/github/AlexMomot-717/python-project-83/test_coverage)
+
+### Setup:
+
```

Comment:
usually, it goes from the steps that need to be taken and then to the expected result

here you start from the expected result and then list all the steps

Reply:
setup block should've stayed at the same place, this comment was about it's content :)